website/sidebars.json

```diff
@@ -81,14 +81,89 @@
       "the-new-architecture/using-codegen",
       "the-new-architecture/codegen-cli"
     ],
-    "Native Modules": [
-      "the-new-architecture/pure-cxx-modules",
-      "the-new-architecture/custom-cxx-types",
-      "the-new-architecture/create-module-library"
-    ],
-    "Native Components": [
-      "the-new-architecture/direct-manipulation-new-architecture",
-      "the-new-architecture/layout-measurements"
+    "Native Development": [
+      {
+        "type": "doc",
+        "id": "native-platform",
+        "label": "Introduction"
+      },
+      {
+        "type": "category",
+        "label": "Modules",
+        "collapsible": false,
+        "collapsed": false,
+        "items": [
+          {
+            "type": "doc",
+            "id": "turbo-native-modules-introduction",
+            "label": "Introduction"
+          },
+          {
+            "type": "doc",
+            "id": "turbo-native-modules-android",
+            "label": "Android"
+          },
+          {
+            "type": "doc",
+            "id": "turbo-native-modules-ios",
+            "label": "iOS"
+          },
```

Comment:
I was looking at the rendered website and we should remove these two. Otherwise they appear in the sidebar but they are partial guides. Those fragments make sense in the context of the initial guide.